packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -281,17 +291,16 @@ export class GCFBootstrapper {
       // Job was scheduled for a single repository:
       await this.scheduledToTask(body.repo, id, body, eventName, signature);
     } else {
-      const octokit = new Octokit({
-        auth: await this.getInstallationToken(body.installation.id),
-      });
+      const octokit = await this.getAuthenticatedOctokit(body.installation.id);
       // Installations API documented here: https://developer.github.com/v3/apps/installations/
-      const installationsPaginated = octokit.paginate.iterator({
-        url: '/installation/repositories',
```

Comment:
@gr2m does this look like a reasonable update to the logic?

Reply:
yes 👍🏼